scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Packaging scripts
+
+This directory contains useful packaging scripts.
+
+## `configure-hypervisor.sh`
+
+This script generates the official set of hypervisor build configuration
```

Comment:
Hi. given the script generates QEMU, and only QEMU, arguments right now, maybe we could be more explicit about that in the text.
For instance, I ran (for a laugh...) `./configure-hypervisor.sh graham`, and was surprised it ran and generated output :-)
Given we are not filtering on the name, and the output is very QEMU specific, I reckon we can be more explicit. you think?